docs/forcing_password_reset.md

```diff
@@ -0,0 +1,67 @@
+# Forcing Password Reset
+
+Depending on the scope of your application, there may be times when you'll decide that it is absolutely necessary to force user(s) to reset their password. This practice is common when you find out that users of your application do not use strong passwords OR there is a reasonable suspicion that their passwords have been compromised. This guide provides you with ways to achieve this.
+
+- [Forcing Password Reset](#forcing-password-reset)
+  - [Available Methods](#available-methods)
+    - [Check if a User Requires Password Reset](#check-if-a-user-requires-password-reset)
+    - [Force Password Reset On a User](#force-password-reset-on-a-user)
+    - [Removing Password Reset Flag On a User](#removing-password-reset-flag-on-a-user)
+    - [Force Password Reset On Multiple Users](#force-password-reset-on-multiple-users)
+    - [Force Password Reset On All Users](#force-password-reset-on-all-users)
+
+## Available Methods
+
+Shield provides a way to enforce password resets throughout your application. The `Resettable` trait on the `User` entity and the `UserIdentityModel` provides the following methods to do so.
+
+### Check if a User Requires Password Reset
+
+When you need to check if a user requires password reset, you can do so using the `requiresPasswordReset()` method on the `User` entity. Returns boolean `true`/`false`.
+
+```php
+if ($user->requiresPasswordReset()) {
+    //...
+}
+```
+
+### Force Password Reset On a User
+
+To force password reset on a user, you can do so using the `forcePasswordReset()` method on the `User` entity.
+
+```php
+$user->forcePasswordReset();
+```
+
+### Remove Force Password Reset Flag On a User
+
+Undoing or removing the force password reset flag on a user can be done using the `undoForcePasswordReset()` method on the `User` entity.
+
+```php
+$user->undoForcePasswordReset();
+```
+
+### Force Password Reset On Multiple Users
+
+If you see the need to force password reset for more than one user, the `forceMultiplePasswordReset()` method of the `UserIdentityModel` allows you to do this easily. It accepts an `Array` of user IDs.
+
+```php
+use CodeIgniter\Shield\Models\UserIdentityModel;
+...
+...
+...
```

Comment:
```suggestion

// ...
```
I understand the purpose, but this is PHP block.

Reply:
This PR is special. No modification is acceptable, because beta.4 was already released.
This PR just merges beta.4 code into `master` branch.

If you want to change any content, send a normail PR to `develop` branch.